src/tcp_wrap.cc

```diff
@@ -428,7 +428,7 @@ void TCPWrap::Connect(const FunctionCallbackInfo<Value>& args) {
 
   assert(args[0]->IsObject());
   assert(args[1]->IsString());
-  assert(args[2]->Uint32Value());
+  assert(args[2]->Uint32Value() >= 0);
```

Comment:
I may be missing something: how can `Uint32Value()` return a value that is not `>= 0`?